test/unit/rules/require-button-type-test.js

```diff
@@ -142,18 +140,19 @@ generateRuleTests({
       fixedTemplate: '<form><button type="submit"></button></form>',
     },
     {
-      template: '/** silly example <button> usage */ <button><button>',
+      template: '<template>/** silly example <button/> usage */ <button></button></template>',
```

Comment:
```suggestion
      template: '<template>/** silly example <button> usage */ <button></button></template>',
```

I think the `button` in the comment probably shouldn't self-close

Reply:
I rebid on your branch upstream, good call.